mmv1/products/notebooks/api.yaml

```diff
@@ -281,6 +281,37 @@ objects:
               Defines whether the instance has the vTPM enabled.
               Enabled by default.
             default_value: true
+      - !ruby/object:Api::Type::Enum
+        name: 'nicType'
+        description: |
+          The type of vNIC driver.
+        values:
+          - UNSPECIFIED_NIC_TYPE
```

Comment:
```suggestion
```

Reply:
This is a default that is not standard. UNSPECIFIED_NIC_TYPE vs NIC_TYPE_UNSPECIFIED. This is because we use Compute API under the hood and decided to use same default value. https://cloud.google.com/workflows/docs/reference/googleapis/compute/v1/Overview (Search for UNSPECIFIED_NIC_TYPE)